generators/spa/templates/Vue3JS/common/packages/Microsoft.AspNet.WebApi.WebHost.5.2.3/lib/net45/System.Web.Http.WebHost.xml

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alejoroman0605 Why isn't this using the same build files as the other module templates?